### ISSUE TITLE
Command delay not explained

### ISSUE THREAD

**Issue by tara81** [Codisimus/PhatLoots#65]
How do i make a command delay?

is it /loot add cmd delay <seconds> /say hi
is it /loot add cmd delay <minutes> /say hi

is it /loot add cmd delay <hours><minutes><seconds> <world/ignore if all worlds> /say hi

?

Your /loot add help only says /loot add cmd [parameter1] [parameter2]

**Comment by Codisimus** [comment]
There is currently no way to add a delay to a command in-game. You must modify the PhatLoot save file and add "Delay: 1200" to the CommandLoot you wish to delay. The number is equal to the number of ticks so 1200 = 1 minute.

**Comment by tara81** [comment]
Is this how you would do it?

ma1:
  ==: PhatLoot
  AutoLoot: false
  BreakAndRespawn: false
  Global: false
  LootList:
  - ==: Money
    Lower: 0
    Probability: 100.0
    Upper: 0
  - ==: Experience
    Lower: 0
    Probability: 100.0
    Upper: 0
  - ==: Command
    Command: bc &0[&bmobarena&0] &7/ma j 1
    FromConsole: true
    Probability: 100.0
    Delay: 1200 #1 minute = 1200 ticks
    TempOP: false
  Name: ma1
  Reset:
    Hours: 0
    Days: 0
    Seconds: 30
    Minutes: 0
  RoundDownTime: false

**Comment by Codisimus** [comment]
Yes, that looks right. You can edit the loot table and use the '/loot reload' command.

I am working on adding a command to make this easier BTW. But I wanted to first give at least basic functionality so that you could begin using it.

**Comment by tara81** [comment]
Codi this doesnt work neither does the network booster permission work properly. I made tickets for both.
